semantics/src/builtins.rs

```diff
@@ -1,5 +1,53 @@
-pub const SELF: &str = "self";
-pub const SENDER: &str = "sender";
-pub const MSG: &str = "msg";
-pub const CLONE: &str = "clone";
-pub const TO_MEM: &str = "to_mem";
+use strum::EnumString;
```

Comment:
Oh, that's cool! I didn't know about `EnumString` and yeah it makes total sense to use enums here for added guidance from the compiler :tada: